tests/CategoryAndArticles.js

```diff
@@ -44,160 +38,154 @@ describe("Category and articles for category", () => {
+
     await HomePage.backPage();
     await expect(await browser).toHaveUrl("https://astage.honesttopaws.com/");
 
     await (await HomePage.goodArticle3).waitForDisplayed();
     await (await HomePage.goodArticle3).click();
-    await Menu.waitForExistHeader();
     await (await AllArticles.galleryCategory).waitForDisplayed();
     await expect(await AllArticles.galleryCategory).toHaveText("Good");
+
     await HomePage.backPage();
     await expect(await browser).toHaveUrl("https://astage.honesttopaws.com/");
 
     await (await HomePage.goodArticle4).waitForDisplayed();
     await (await HomePage.goodArticle4).click();
-    await Menu.waitForExistHeader();
     await (await AllArticles.galleryCategory).waitForDisplayed();
     await expect(await AllArticles.galleryCategory).toHaveText("Good");
-    await HomePage.backPage();
-    await expect(await browser).toHaveUrl("https://astage.honesttopaws.com/");
-
-    await (await HomePage.seeMoreGoodeButton).waitForDisplayed();
-    await (await HomePage.seeMoreGoodeButton).click();
-    await Menu.waitForExistHeader();
-    await expect(await Menu.header).toHaveText("GOOD");
   });
 
-  it("Articles for omg category", async () => {
-    await (await HomePage.omgArticle1).waitForDisplayed();
+  it("Click on all articles for omg category", async () => {
+    await await HomePage.omgArticle1.waitForDisplayed();
     await (await HomePage.omgArticle1).click();
-    await Menu.waitForExistHeader();
     await (await AllArticles.galleryCategory).waitForDisplayed();
     await expect(await AllArticles.galleryCategory).toHaveText("Omg");
+
     await HomePage.backPage();
-    await Menu.waitForExistHeader();
     await expect(await browser).toHaveUrl("https://astage.honesttopaws.com/");
 
     await (await HomePage.omgArticle2).waitForDisplayed();
     await (await HomePage.omgArticle2).click();
-    await Menu.waitForExistHeader();
     await (await AllArticles.galleryCategory).waitForDisplayed();
     await expect(await AllArticles.galleryCategory).toHaveText("Omg");
+
     await HomePage.backPage();
-    await Menu.waitForExistHeader();
     await expect(await browser).toHaveUrl("https://astage.honesttopaws.com/");
 
+    await await HomePage.omgArticle3.waitForDisplayed();
     await (await HomePage.omgArticle3).click();
-    await Menu.waitForExistHeader();
     await (await AllArticles.galleryCategory).waitForDisplayed();
     await expect(await AllArticles.galleryCategory).toHaveText("Omg");
+
     await HomePage.backPage();
-    await Menu.waitForExistHeader();
     await expect(await browser).toHaveUrl("https://astage.honesttopaws.com/");
 
     await (await HomePage.omgArticle4).waitForDisplayed();
     await (await HomePage.omgArticle4).click();
-    await Menu.waitForExistHeader();
     await (await AllArticles.galleryCategory).waitForDisplayed();
     await expect(await AllArticles.galleryCategory).toHaveText("Omg");
+  });
+
+  it("Click on all see more button", async () => {
```

Comment:
This title is not so easy to understand. You specify that you click on all "See more button". There is only one see more button? There is a chrome extension that I think will help you for English writing: [Grammarly for Chrome](https://chrome.google.com/webstore/detail/grammarly-for-chrome/kbfnbcaeplbcioakkpcpgfkobkghlhen)

Reply:
No, there are more buttons. I did not realize